File: spring_java_orm_sample/src/main/java/com/example/demo/doma/entity/BookStockWithStoreName.java
package com.example.demo.doma.entity;

import lombok.Getter;
import lombok.Setter;
import org.seasar.doma.Column;
import org.seasar.doma.Entity;
import org.seasar.doma.Id;

@Entity
@Getter
@Setter
public class BookStockWithStoreName {
    @Id
    @Column(name = "id")
    Long id;

    @Column(name = "book_stock_store_id")
    Long bookStockStoreId;

    @Column(name = "store_name")
    String storeName;

    @Column(name = "book_stock_quantity")
    Integer bookStockQuantity;
}

File: codex_poting/kotlin/src/main/kotlin/com/example/demo/doma/entity/BookStockWithStoreName.kt
package com.example.demo.doma.entity

import org.seasar.doma.Column
import org.seasar.doma.Entity
import org.seasar.doma.Id

@Entity
class BookStockWithStoreName {
    @Id
    @Column(name = "id")
    var id: Long? = null

    @Column(name = "book_stock_store_id")
    var bookStockStoreId: Long? = null

    @Column(name = "store_name")
    var storeName: String? = null

    @Column(name = "book_stock_quantity")
    var bookStockQuantity: Int? = null
}
